app/contributing/plugin-docs.md

```diff
@@ -44,6 +43,42 @@ should be a square-format PNG file, 120x120 pixels in size.
 
     Add the icon file to `/app/_assets/images/icons/hub/`. 
 
+### Third-party or partner plugins
+
+1. Set up or find your publisher directory in the [docs repository](https://github.com/Kong/docs.konghq.com/tree/main/app/_hub/).
+
+   If your company has not contributed a plugin to Kong before, 
+   create a publisher directory at`_app/_hub/`, such as
+  `_app/_hub/company-name`. 
+
+    See other Kong Hub listings for examples of publisher names.
+
+1. Create a subdirectory for your plugin within your publisher directory.
```

Comment:
```suggestion
1. Create a subdirectory for your plugin within the publisher directory.
```

This one feels like `the` the because the directory belongs to kong.